### PR TITLE
Shifted the png files of eip-823 from root to assets

### PR DESCRIPTION
**Pull request by kashishkhullar**

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - File assets/eip-823/eip-823-token-exchange-standard-visual-representation-1.PNG is not an EIP
 - File assets/eip-823/eip-823-token-exchange-standard-visual-representation-2.PNG is not an EIP

**Comment by Arachnid** [comment]
Thanks!

File: EIPS/eip-823.md [hunk 1]
```diff
@@ -187,13 +187,13 @@ event ExchangeAndSpent ( address _from, address _by, uint _value ,address _targe
...
 #### Exchanging Tokens
-![token-exchange-standard-visual-representation-1](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/EIPS/token-exchange-standard-visual-representation-1.PNG)
+![token-exchange-standard-visual-representation-1](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/assets/eip-823/eip-823-token-exchange-standard-visual-representation-1.PNG)
 
 NOTE: After the successful exchange the contract on right owns some tokens of the contract on the left.
 
 #### Exchanging And Spending Tokens
 
-![token-exchange-standard-visual-representation-2](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/EIPS/token-exchange-standard-visual-representation-2.PNG)
+![token-exchange-standard-visual-representation-2](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/assets/eip-823/eip-823-token-exchange-standard-visual-representation-2.PNG)
```
Comment (axic) [inline EIPS/eip-823.md]: @kashishkhullar you have added the files to the `assets` directory, but these links are pointing to a remote repo. Can you change these links to be `.. /assets/eip-823/eip-823-token-exchange-standard-visual-representation-2.PNG` ?

Comment (kashishkhullar) [reply EIPS/eip-823.md]: I have updated the links.

Comment (axic) [reply EIPS/eip-823.md]: Thanks!

**Pull request merged**